Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -1868,6 +1868,20 @@ CaretBrowsingEnabled:
     WebCore:
       default: false
 
+CaretPositionFromPointEnabled:
+  type: bool
+  status: testable
+  category: dom
+  humanReadableName: "CaretPositionFromPoint API"
```

Comment:
Shouldn't this be `document.caretPositionFromPoint()` or some such?

Reply:
Yes, that is more specific, fixed.